nextflow.config

```diff
@@ -129,6 +129,13 @@ params {
     skip_deduplication = false
     deduplication_tool = 'markduplicates'
 
+    // Contamination estimation
+    run_contamination_angsd = false
+    angsd_chrom_name        = 'X'
+    angsd_mapq              = 30
+    angsd_minq              = 30
+    angsd_hapmap            = "${baseDir}/assets/angsd_resources/HapMapChrX.gz"
```

Comment:
Parameters should be named `run_contamination_estimation_angsd` &`contamination_estimation_angsd_*` to follow schema.